packages/main/src/themes/SegmentedButton.css

```diff
@@ -50,30 +57,60 @@
 
 ::slotted([ui5-segmented-button-item][pressed]:nth-child(odd)),
 ::slotted([ui5-segmented-button-item][active]:nth-child(odd)) {
-	border-inline-end: var(--_ui5_segmented_btn_inner_pressed_border_odd_child);
-	border-inline-start: var(--_ui5_segmented_btn_inner_pressed_border_odd_child);
+	box-shadow: var(--_ui5_segmented_btn_item_pressed_active_ODD_box_shadow);
```

Comment:
--sapButton_Active_BorderColor parameter is not used, but it is present in the spec.
Instead there is the following parameter: _ui5_segmented_btn_item_pressed_active_box_shadow, which corresponds to --sapButton_Selected_BorderColor instead.